freestyle/shared/src/main/scala/freestyle/module.scala

```diff
@@ -166,8 +166,9 @@ final class moduleImpl(val c: Context) {
       case q"$mods val $name: $tpt[..$args]" => q"$mods val $name: $tpt[$FF, ..$args]"
       case x => x
     }
+    val parentTrees = parents ++ List(tq"$FreeModuleLike", tq"freestyle.EffectLike[$FF]")
```

Comment:
should these be fully qualified to preserve macro hygiene? `_root_.freestyle.FreeModuleLike` and `_root_.freestyle.EffectLike[$FF]` ?